### PR TITLE
Chore updating xray examples

### PR DESCRIPTION
**Pull request by xoscar**
This PR updates the xray examples

## Changes

- updates the x-ray examples

## Fixes

- https://github.com/kubeshop/tracetest-cloud/issues/807

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by jfermi — APPROVED** [review]

File: docs/docs/examples-tutorials/recipes/running-tracetest-with-aws-x-ray.mdx [hunk 1]
```diff
@@ -91,276 +78,38 @@ The Node.js app is a simple Express app, contained in [the `index.js` file](http
...
-You will need [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed on your machine to run this quick start app!
-
-And a set of AWS credentials to connect Tracetest to the cloud API.
-
-## Project Structure
-
-The project is built with Docker Compose.
-
-### 1. Node.js App
+The [`docker-compose.yaml` file](https://github.com/kubeshop/tracetest/blob/main/examples/tracetest-amazon-x-ray/docker-compose.agent.yaml) and [`Dockerfile`](https://github.com/kubeshop/tracetest/blob/main/examples/tracetest-amazon-x-ray/Dockerfile) in the root directory contain the Node.js app.
```
Comment (adnanrahic) [inline docs/docs/examples-tutorials/recipes/running-tracetest-with-aws-x-ray.mdx:96]:
```suggestion
The [`docker-compose.yaml` file](https://github.com/kubeshop/tracetest/blob/main/examples/tracetest-amazon-x-ray/docker-compose.yaml) and [`Dockerfile`](https://github.com/kubeshop/tracetest/blob/main/examples/tracetest-amazon-x-ray/Dockerfile) in the root directory contain the Node.js app.
```

File: docs/docs/examples-tutorials/recipes/running-tracetest-with-aws-x-ray-pokeshop.mdx [hunk 2]
```diff
@@ -156,338 +105,19 @@ The [`docker-compose.agent.yaml` file](https://github.com/kubeshop/tracetest/blo
 
 The [`docker-compose.agent.yaml` file](https://github.com/kubeshop/tracetest/blob/main/examples/tracetest-amazon-x-ray-pokeshop/docker-compose.agent.yaml) also contains the Tracetest Agent and ADOT.
```
Comment (adnanrahic) [inline docs/docs/examples-tutorials/recipes/running-tracetest-with-aws-x-ray-pokeshop.mdx:106]:
```suggestion
The [`docker-compose.yaml` file](https://github.com/kubeshop/tracetest/blob/main/examples/tracetest-amazon-x-ray-pokeshop/docker-compose.yaml) also contains the Tracetest Agent and ADOT.
```

Comment (adnanrahic) [reply docs/docs/examples-tutorials/recipes/running-tracetest-with-aws-x-ray-pokeshop.mdx:106]: Also edit the links on line 104 above.

File: docs/docs/examples-tutorials/recipes/running-tracetest-with-aws-x-ray-adot.mdx [hunk 3]
```diff
@@ -95,296 +82,38 @@ The Node.js app is a simple Express app, contained in [the `index.js` file](http
...
-```
-
-```bash
-tracetest config -t <YOUR_API_TOKEN>
-tracetest apply datastore -f ./tracetest-tracing-backend.yaml
-```
-
 ## Run the Node.js App, ADOT, and Tracetest Agent with Docker Compose
 
 The [`docker-compose.agent.yaml` file](https://github.com/kubeshop/tracetest/blob/main/examples/tracetest-amazon-x-ray-adot/docker-compose.agent.yaml) and [`Dockerfile`](https://github.com/kubeshop/tracetest/blob/main/examples/tracetest-amazon-x-ray-adot/Dockerfile) in the root directory contain the Node.js app.
```
Comment (adnanrahic) [inline docs/docs/examples-tutorials/recipes/running-tracetest-with-aws-x-ray-adot.mdx:100]:
```suggestion
The [`docker-compose.yaml` file](https://github.com/kubeshop/tracetest/blob/main/examples/tracetest-amazon-x-ray-adot/docker-compose.yaml) and [`Dockerfile`](https://github.com/kubeshop/tracetest/blob/main/examples/tracetest-amazon-x-ray-adot/Dockerfile) in the root directory contain the Node.js app.
```

File: docs/docs/examples-tutorials/recipes/running-tracetest-with-aws-x-ray-adot.mdx [hunk 4]
```diff
@@ -95,296 +82,38 @@ The Node.js app is a simple Express app, contained in [the `index.js` file](http
...
 
 The [`docker-compose.agent.yaml` file](https://github.com/kubeshop/tracetest/blob/main/examples/tracetest-amazon-x-ray-adot/docker-compose.agent.yaml) also contains the Tracetest Agent and ADOT.
```
Comment (adnanrahic) [inline docs/docs/examples-tutorials/recipes/running-tracetest-with-aws-x-ray-adot.mdx:102]:
```suggestion
The [`docker-compose.yaml` file](https://github.com/kubeshop/tracetest/blob/main/examples/tracetest-amazon-x-ray-adot/docker-compose.yaml) also contains the Tracetest Agent and ADOT.
```

File: docs/docs/examples-tutorials/recipes/running-tracetest-with-aws-x-ray.mdx [hunk 5]
```diff
@@ -91,276 +78,38 @@ The Node.js app is a simple Express app, contained in [the `index.js` file](http
...
 
-The `Dockerfile` in the root directory is for the Node.js app.
+The [`docker-compose.yaml` file](https://github.com/kubeshop/tracetest/blob/main/examples/tracetest-amazon-x-ray/docker-compose.agent.yaml) also contains the Tracetest Agent and X-Ray Daemon.
```
Comment (adnanrahic) [inline docs/docs/examples-tutorials/recipes/running-tracetest-with-aws-x-ray.mdx:98]:
```suggestion
The [`docker-compose.yaml` file](https://github.com/kubeshop/tracetest/blob/main/examples/tracetest-amazon-x-ray/docker-compose.yaml) also contains the Tracetest Agent and X-Ray Daemon.
```

**Review by adnanrahic — APPROVED** [review]
A few links need fixing. All good to merge otherwise.

**Pull request merged**